feature/home/src/main/java/com/school_of_company/home/util/QrGenerator.kt

```
@@ -0,0 +1,91 @@
+
+            val encodeHints = mutableMapOf<EncodeHintType, Any?>()
+                .apply {
+                    this[EncodeHintType.MARGIN] = padding
+                }
+
+            val bitmapMatrix = try {
+                qrCodeWriter.encode(
+                    content, BarcodeFormat.QR_CODE,
+                    size, size, encodeHints
+                )
+            } catch (ex: WriterException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)